app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
oh, possibly, but more importantly, if they right-click on the tile, in addition to things like:

Copy <hex>
Copy #<hex>
Copy rgba(...)

they'd see:

Copy MATERIAL_COLOR...
Copy $material-color(...

and whatever other formats they've defined